### ISSUE TITLE
How to have one InferenceCalculator run twice in a graph

### ISSUE THREAD

**Issue by tri-bao** [google/mediapipe#4879]
The InferenceCalculator only accepts one input tensor stream and has one output tensor stream. In my graph, I have 2 input tensor streams and want to get 2 corresponding output tensor streams from the InferenceCalculator. In normal program, if the inference calculator is just a `forward` method, I would just call the `forward` method 2 times, one after one in order to get 2 output tensors. How you could I do that with mediapipe graph?

To demonstrate the issue, my graph looks like
![image](https://github.com/google/mediapipe/assets/11454377/dbeb6639-2d81-4613-a621-0d98696a40a8)

If I duplicate the InferenceCalculator into 2 nodes (loading same model), it works but the graph load time will be double and it make our application unusable.

**Comment by schmidt-sebastian** [comment]
I don't think this is possible today. You might be able to combine the tensors into a single stream if you can adjust your models. But otherwise you would have to run two calculators, which each need to load their own model. Sorry that we do not have a better solution for you at this point.

**Comment by tri-bao** [comment]
I found myself the solution to this problem with mediapipe. Sharing it here in case anybody needs it: To achieve this goal we can use the pair of calculators: `BeginLoopCalculator` and `EndLoopCalculator`
